filebeat-yml-script.py

```diff
@@ -33,6 +32,7 @@ def get_log_level():
 logzio_region = os.getenv("LOGZIO_REGION", "")
 logzio_codec = os.getenv("LOGZIO_CODEC", "plain").lower()
 logzio_codec_list = ["plain", "json"]
+input_encoding = os.getenv("INPUT_ENCODING", "utf-8")
```

Comment:
Please add a check that the customer set the right encoding
Also please keep in mind - big /small characters

Reply:
Done.